worker.go

```diff
@@ -288,10 +290,9 @@ func doWork(
 		return problems.Add("failed to list managed stacks: %w", err)
 	}
 
-	for _, stack := range stacks {
-		if err := stack.Err(); err != nil {
-			problems.Add("stack %s error: %w", stack.Name, err)
-		}
```

Comment:
I think we need to keep this loop to collect stack errors into problem list

Reply:
yeah true, will fix it